dev/com.ibm.ws.microprofile.reactive.messaging.kafka.adapter/src/com/ibm/ws/microprofile/reactive/messaging/kafka/adapter/KafkaAdapterFactory.java

```diff
@@ -29,7 +33,7 @@
     private static final Class<?>[] KAFKA_CONSUMER_ARG_TYPES = { Map.class };
 
     private static final String KAFKA_PRODUCER_IMPL = "com.ibm.ws.microprofile.reactive.messaging.kafka.adapter.impl.KafkaProducerImpl";
-    private static final Class<?>[] KAFKA_PRODUCER_ARG_TYPES = { Map.class };
+    private static final Class<?>[] KAFKA_PRODUCER_ARG_TYPES = { Map.class, boolean.class }; //TODO remove beta guard before GA
```

Comment:
Instead of passing beta guard booleans around, can we put a static somewhere in the interface bundle?

Reply:
Done